Add Sector Shock Bar Chart Comparison

ylyangtw opened the pull request:
Need help for coloring...
![image](https://github.com/IN-CORE/incore-playbook/assets/63822845/fc139cbf-cfb7-46de-8185-d91dd4e60197)

ylyangtw committed:
Add Sector Shock Bar Chart Comparison

ylyangtw committed:
Update SectorShocksBarChartCompare.jsx

longshuicy commented on src/components/childComponents/graph/SectorShocksBarChartCompare.jsx:
```diff
@@ -0,0 +1,136 @@
+					legend: { position: "top", alignment: "center" },
+					series: {
+					  0: { color: "#2d8686" }, // Color for the first data series
+					  1: { color: "#F1C40F" }  // Color for the second data series
```
This legend color is confusing. Maybe I would leave it out.

longshuicy commented on src/components/childComponents/graph/SectorShocksBarChartCompare.jsx:
```diff
@@ -0,0 +1,136 @@
+const ParseData = (dataA, dataB) => {
+	const chartData = [];
+	const colorsA = ["#aededb", "#2d8686", "#00619D", "#2c698d", "#272643"];
+	const colorsB = ["#F9E79F", "#F1C40F", "#7D6608", "#784212", "#641E16"];
```
I plan to ask the designer for a final review and critique. For now, how about 
```
const colorsB = ["#c1f0e5", "#3daaaa", "#0080bf", "#3e8caa", "#34385a"];
```

<img width="1459" alt="image" src="https://github.com/IN-CORE/incore-playbook/assets/13950475/44389e5b-d473-4c23-9bff-13a63d252b4d">

longshuicy commented on src/components/childComponents/graph/SectorShocksBarChartCompare.jsx:
```diff
@@ -0,0 +1,136 @@
+					  0: { color: "#2d8686" }, // Color for the first data series
+					  1: { color: "#F1C40F" }  // Color for the second data series
+					},
+					height: "25em",
```
You might be able to increase the height since there is no other content on the page

ylyangtw committed:
changed colorsB, remove legends, changed height

ylyangtw commented:
@longshuicy Thanks much! I've addressed the feedbacks

![image](https://github.com/IN-CORE/incore-playbook/assets/63822845/129301f7-5091-42ac-8327-c0d84615f4f5)

longshuicy committed:
Merge branch '277-compare-multiple-runs' into 305-compare-sector-shock-bar-chart
